com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/persistence/ProxyModelAssociationsAdapter.java

```diff
@@ -69,6 +70,8 @@ private void ensureAssociationsLoaded() {
       if (resource.eAdapters().remove(this)) {
         DirectLinkingResourceStorageLoadable loadable = (DirectLinkingResourceStorageLoadable) ((DirectLinkingResourceStorageFacade) resource.getResourceStorageFacade()).getOrCreateResourceStorageLoadable(resource);
         try {
+          // the associations are mappings from AST model elements to inferred model elements (and back) so we cannot skip loading the model.
+          ResourceSetOptions.setSkipModel(resource.getResourceSet(), false);

```

Comment:
should we log a warning?

Reply:
I don't think so. Isn't this just a load that was missed when the 'skipModel' option was added?

Reply:
but that is a programming error, isn't it?